src/bindings/c/include/openvino/c/ov_node.h

```diff
@@ -35,70 +44,93 @@ typedef struct {
  */
 
 /**
- * @brief Get the shape of ov_output_node.
+ * @brief Get the shape of port object.
+ * @ingroup node
+ * @param port A pointer to ov_output_const_node_t.
+ * @param tensor_shape tensor shape.
+ * @return Status code of the operation: OK(0) for success.
+ */
+OPENVINO_C_API(ov_status_e) ov_const_node_get_shape(ov_output_const_node_t* port, ov_shape_t* tensor_shape);
+
+/**
+ * @brief Get the shape of port object.
  * @ingroup node
- * @param nodes A pointer to ov_output_const_node_t.
+ * @param port A pointer to ov_output_node_t.
  * @param tensor_shape tensor shape.
  * @return Status code of the operation: OK(0) for success.
  */
-OPENVINO_C_API(ov_status_e) ov_node_get_shape(ov_output_const_node_t* node, ov_shape_t* tensor_shape);
+OPENVINO_C_API(ov_status_e) ov_node_get_shape(ov_output_node_t* port, ov_shape_t* tensor_shape);
 
 /**
- * @brief Get the tensor name of ov_output_node list by index.
+ * @brief Get the tensor name of port list by index.
  * @ingroup node
- * @param nodes A pointer to the ov_output_node_list_t.
- * @param idx Index of the input tensor
+ * @param port_list A pointer to the ov_output_const_node_list_t.
+ * @param idx Index in the port_list.
  * @param tensor_name A pointer to the tensor name.
  * @return Status code of the operation: OK(0) for success.
  */
 OPENVINO_C_API(ov_status_e)
-ov_node_list_get_any_name_by_index(ov_output_node_list_t* nodes, size_t idx, char** tensor_name);
+ov_node_list_get_any_name_by_index(ov_output_const_node_list_t* port_list, size_t idx, char** tensor_name);
 
 /**
- * @brief Get the shape of ov_output_node.
+ * @brief Get the shape of port by index.
  * @ingroup node
- * @param nodes A pointer to the ov_output_node_list_t.
- * @param idx Index of the input tensor
- * @param tensor_shape tensor shape.
+ * @param port_list A pointer to the ov_output_const_node_list_t.
+ * @param idx Index in the port_list.
+ * @param shape The shape of the port.
  * @return Status code of the operation: OK(0) for success.
  */
 OPENVINO_C_API(ov_status_e)
-ov_node_list_get_shape_by_index(ov_output_node_list_t* nodes, size_t idx, ov_shape_t* shape);
+ov_node_list_get_shape_by_index(ov_output_const_node_list_t* port_list, size_t idx, ov_shape_t* shape);
```

Comment:
I propose:
```suggestion
ov_output_node_get_shape(ov_output_node_t* port, ov_shape_t* shape);
```

Reply:
How to distinguish  `ov::Output<ov::Node>` and `<ov::Output<const ov::Node>`?